### ISSUE TITLE
Dissect ITCH feed delivered over SoupBinTCP

### ISSUE THREAD

**Issue by chutzimir** [Open-Markets-Initiative/wireshark-lua#49]
Please consider adding support for ITCH feed delivered over SoupBinTCP. The current dissector only supports feeds delivered over MoldUDP64, for the JNX dissectors at least.

The ITCH feed is also supported over SoupBinTCP.

Furthermore, the [GLIMPSE protocol](https://www.japannext.co.jp/pub_data/pub_onboarding/Japannext_PTS_GLIMPSE_Equities_v1.3.pdf) is essentially a point-in-time snapshot of the market delivered over ITCH, but only delivered over SoupBinTCP. Supporting ITCH-over-SoupBinTCP would automatically add support for GLIMPSE.

The OUCH protocol dissector already understands the SoupBinTCP packet structure, so the work is already done somewhere. Merging these together, may not be so trivial.

**Comment by Open-Markets-Initiative** [comment]
Now that we have a pcap, Tcp Itch is simple.  Our binary models was designed to handle Glimpse, however it might take a week or more to work out the details.  We have a few other things on the list, but now that we have a pcap we can circle back and get that operational.

**Comment by Open-Markets-Initiative** [comment]
We are going to do this in parallel with another source generation project.

**Comment by Open-Markets-Initiative** [comment]
569d2ce98b5be026f0bcb6a218990784c18b4f82 for Tcp market data.  I need to think about Glimpse more
